### ISSUE TITLE
[bug] Select - floating-ui autoUpdate runs continuously after closing select

### ISSUE THREAD

**Issue by nderscore** [tamagui/tamagui#519]
Last seen on: 1.0.4

**Describe the bug**

In the web implementation of the `Select` component, after opening the select once and closing it, `floating-ui`'s `autoUpdate()` function gets called continuously from that point onwards, despite the select's popover being closed.

**To Reproduce**

1. Open any page with a `<Select>` component on it, such as the [docs page for the `Select` component](https://tamagui.dev/docs/components/select/1.0.0).

1. Run this snippet in devtools console to add logging to all `requestAnimationFrame`/`cancelAnimationFrame` calls:

```js
(() => {
  const RAF = window.requestAnimationFrame;
  // just in case, get native cancelAnimationFrame implementation:
  const iframe = document.createElement('iframe');
  iframe.style.display = 'none';
  document.body.appendChild(iframe);
  const CAF = iframe.contentWindow.cancelAnimationFrame;

  // add logging wrappers:
  window.requestAnimationFrame = (callback) => {
    const frame = RAF(callback);
    console.log('RAF', frame, callback);
    return frame;
  };

  window.cancelAnimationFrame = (frame) => {
    console.log('CAF', frame);
    return CAF(frame);
  };
})();
```

1. Open the select

1. Close the select

1. See `autoUpdate` still getting called continuously

**Comment by natew** [comment]
Let me know if this improved it: https://github.com/tamagui/tamagui/commit/cf2951a6c6aab70d1361e71c20126f725407fec9 out in 1.0.5

**Comment by nderscore** [comment]
> Let me know if this improved it

It didn't seem to make any difference 😭  ...but I think I found the solution: #525 (this also fixes the issue I saw in #514 with the scroll buttons never terminating)
